contributing.md

```diff
@@ -5,3 +5,63 @@
 https://www.conventionalcommits.org/en/v1.0.0/
 
 Commits are linted via using [commitlint](https://github.com/conventional-changelog/commitlint)
+
+## Before you open the PR
+
+This little step-by-step guide will ensure you have the latest changes!
+
+### Step 1.
+
+If you don't already have the main repo of prytaneum as a remote repo, do the following:
+
+```bash
+git remote add upstream git@github.com:c2c-project/prytaneum.git
```

Comment:
Maybe just specify that the remote repo can be named anything and `upstream` is just one possibility.

Reply:
Will do

Reply:
fixed.